english/candyland/diabetes.md

```diff
@@ -0,0 +1,20 @@
+You light a small fire in the middle of the marshmallow room.
+
+As the fire grows, a sweet aroma fills the room and the ceiling becomes ever so slightly brown just above the fire.
+
+The marshmallow room started melting and you started seeing some colorful lights shining through.
+
+Wait! But this is not any colorful lights you are seeing!  What appears before you is a rainbow colored Gumdrop Mountains!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.